editoast/src/models/rolling_stock/light_rolling_stock.rs

```diff
@@ -43,8 +43,8 @@ pub struct LightRollingStockModel {
     rolling_resistance: DieselJson<RollingResistance>,
     #[schema(value_type = LoadingGaugeType)]
     loading_gauge: String,
-    #[schema(value_type = HashMap<String, String>)]
-    power_restrictions: Option<JsonValue>,
+    #[schema(value_type = Option<HashMap<String, String>>)]
```

Comment:
This change should be reflected in the openapi, can you check this model is indeed included in the schemas?

Reply:
```
 power_restrictions:
    additionalProperties:
      type: string
    type: object
```

Reply:
Yeah but you made that field optional, so the `required:` object should be updated. This schema may be missing in the openapi.

Reply:
it was a mistake, I  will keep it as it was before, without `Option`.